### DISCUSSION TITLE
scrolling to the top easily from long list of readme content

### DISCUSSION THREAD

**Discussion by Biniben** [community/community#43071] [Accessibility] [answered]
### Select Topic Area

Product Feedback

### Body

When i open a long list of readme file and scroll down far from top or I use the link for list of content then I want to go back to top but I don't want use "scroll bar" or scroll my mouse instead I want to use 'back to top' icon on right bottom of the screen like other websites has or simpler way of getting to the top of the page. If I make my point right... ?

**Comment by SilasRodrigues19** [comment, marked as answer]
Hi @Biniben 

There is a way to make this button `back to top` manually, if you want, you can see in this other discussion that I answered.

https://github.com/community/community/discussions/42712

**Comment by cehfisher** [comment]
Hi @Biniben - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/43071#discussioncomment-4567071). Could you please close this issue? Thank you! (cc @queenofcorgis)
